src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -932,7 +933,8 @@ internal static SyntaxTrivia FindTriviaByOffset(SyntaxNode node, int textOffset,
                     {
                         if (element.IsNode)
                         {
-                            return FindTriviaByOffset(element.AsNode(), textOffset, stepInto);
```

Comment:
There are two more call sites in this function. Is there a reason to not change them too?